### PR TITLE
Request processing with basic logging and exception handling for http-triggered functions #172

### PR DESCRIPTION
**Pull request by michaelmajgaard**
Request processing with basic logging and exception handling for http-triggered functions #172

### CODE REVIEW

File: source/Energinet.DataHub.PostOffice.Tests/Common/Extensions/HttpRequestDataExtensionsTests.cs [hunk 1]
```diff
@@ -55,5 +58,62 @@ public void CreateResponse_SourceIsNull_ThrowsException()
...
+
+        [Fact]
+        public async Task ProcessAsync_WorkerFinishesSuccessfully_ReturnsResponseFromWorker()
+        {
+            // arrange
+            var request = new MockedHttpRequestData(new MockedFunctionContext()).HttpRequestData;
+            const string? responseData = "Some data";
+
+            // act
+            var actual = await request.ProcessAsync(() => Task.FromResult(request.CreateResponse(new MemoryStream(Encoding.UTF8.GetBytes(responseData))))).ConfigureAwait(false);
```
Comment (FirestarJes) [inline source/Energinet.DataHub.PostOffice.Tests/Common/Extensions/HttpRequestDataExtensionsTests.cs:70]: Should be split to multiple lines

**Review by FirestarJes — CHANGES_REQUESTED** [review]
Der er kun en enkelt lille ting der lige skal kigges på.

File: source/Energinet.DataHub.PostOffice.Tests/Common/Extensions/HttpRequestDataExtensionsTests.cs (hunk 1 above)
Comment (michaelmajgaard) [reply source/Energinet.DataHub.PostOffice.Tests/Common/Extensions/HttpRequestDataExtensionsTests.cs]: Fixed

**Review by FirestarJes — DISMISSED** [review]

**Review by defectiveAi — APPROVED** [review]

**Review by mbusk88 — APPROVED** [review]
See comments

**Pull request merged**